.circleci/config.yml

```diff
@@ -306,10 +305,22 @@ jobs:
             source $BASH_ENV
             make test_sim_gaia_fast
 
+  docker_image:
+    <<: *linux_defaults
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run: |
+          docker build -t tendermint/gaia:$CIRCLE_SHA1 .
```

Comment:
I'd suggest to use the `gaiad version`'s output instead of `$CIRCLE_SHA1`

Reply:
Done.